modules/n1ql/pages/n1ql-intro/queriesandresults.adoc

```diff
@@ -169,15 +169,21 @@ SELECT name, email FROM `users` WHERE name="Monty Python" OR email="monty@python
 Indexes help improve the performance of a query.
 When an index includes the actual values of all the fields specified in the query, the index covers the query and eliminates the need to fetch the actual values from the Data Service.
 An index, in this case, is called a covering index and the query is called a covered query.
-For more information, refer to xref:indexes:covering-indexes.adoc[Covering Indexes].
+
+For more information, refer to xref:learn:services-and-indexes/indexes/global-secondary-indexes.adoc[Using Indexes].
 
 == Index Building
 
-Creating indexes can take a long time on keyspaces with lots of existing documents.
-You can build indexes in the background, creating _deferred_ indexes and then building all deferred indexes at once.
+Index creation happens in two phases: the [def]_creation phase_ and the [def]_build phase_.
+During the creation phase, the Index Service validates the user input, decides the host node for the index, and creates the index metadata on the host node.
+During the build phase, the Index Service reads the documents from the Data Service and builds the index.
```

Comment:
We can mention that the index build happens using DCP.

Reply:
Leaving this for now because we're aiming to bring the Index Lifecycle page and the Index Build information from Queries and Results closer together as part of [DOC-8737](https://issues.couchbase.com/browse/DOC-8737).